src/Components/Buttons.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Button from './Button';
+import styles from 'styled-components';
+
+const Buttons = ({ clear }) => {
+  const Box = styles.div`
+    width: 200px;
+    height: 250px;
+    overflow-y: auto;
+  `;
+  
+  return(
+    <Box>
+      <Button text={'AC'} color={'darkGray'} onClick={clear}/>
+      <Button text={'+/-'} color={'darkGray'}/>
+      <Button text={'%'} color={'darkGray'}/>
+      <Button text={'÷'} color={'orange'}/>
```

Comment:
When the value is a string, you don't have to use `{}`, but simply `<Button text="simplu" />`.
Don't change all these lines now, keep in mind for the future.